orga/blueprints/component-test/files/tests/integration/components/__name___test.js

```diff
@@ -10,7 +10,6 @@ module('Integration | Component |  <%= dasherizedModuleName %>', function (hooks
     // given
 
     //  when
-    await renderScreen(hbs`<<%= classifiedModuleName %> />`);
```

Comment:
Because :
```

blueprints/component-test/files/tests/integration/components/__name___test.js
  1:0  error  Unclosed element `classifiedModuleName`:

|
|  <<%= classifiedModuleName %>
|

(error occurred in 'an unknown module' @ line 1 : column 0)  undefined
```

Reply:
SOLVED